.github/workflows/lint_python.yml

```diff
@@ -1,30 +0,0 @@
-name: lint Python
-on: [pull_request]
-jobs:
-  lint_python:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-python@v3
-      - uses: pre-commit/action@v3.0.0
-      - run: pip install --upgrade pip wheel
-      - run: pip install -U bandit codespell flake8 isort pytest pyupgrade tox
-
-      - name: bandit
-        run: bandit -r . || true
```

Comment:
Bandit has thousands of issues, and we don't care about them anyway.

None of the bandit issues is high risk.